gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRTaskStateTracker.java

```diff
@@ -117,4 +113,40 @@ protected void updateTaskMetrics() {
       this.context.progress();
     }
   }
+  
+  private void updateCounters(Task task) {
+    updateCounters(task, MetricGroupFilter.JOB);
+    updateCounters(task, MetricGroupFilter.TASK);
+  }
+  
+  private void updateCounters(Task task, MetricGroupFilter filter) {
+    Map<String, Counter> counters = JobMetrics.get(null, task.getJobId()).getMetricContext().getCounters(filter);
+    if (counters != null) {
+      for (Map.Entry<String, Counter> entry : counters.entrySet()) {
+        this.context.getCounter(filter.getGroupName(), entry.getKey()).setValue(entry.getValue().getCount());
+      }
+    }
+  }
+  
+  private enum MetricGroupFilter implements MetricFilter {
+    JOB() {
+      @Override
+      public String getGroupName() {
+        return MetricGroup.JOB.name();
```

Comment:
Fixed. I just followed the code style according to this.context.getCounter(MetricGroup.JOB.name().